python/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -177,69 +199,71 @@ def getOutputDataLFNs(id, pipe_out=True):
 
     if pipe_out:
         output(result)
-    return result
+    else:
+        return result
 
 
-def normCPUTime(id):
+def normCPUTime(id, pipe_out=True):
+    ''' Get the normalied CPU time that has been used by a DIRAC job of ID and pipe it out or return it'''
     parameters = dirac.parameters(id)
     ncput = None
     if parameters is not None and parameters.get('OK', False):
         parameters = parameters['Value']
         if 'NormCPUTime(s)' in parameters:
             ncput = parameters['NormCPUTime(s)']
-    output(ncput)
+    if pipe_out:
+        output(ncput)
+    else:
+        return ncput
+
 
+def finalize_job(id, outputDir=os.getcwd(), oversized=True):
```

Comment:
Given my spelling ability in general you're lucky I stuck to a dictionary. I'll change this since it's just in 2 places, but there is a lack of conformity in a lot of other places, and I suspect this is a minor issue